tests/txpool_transactions/get-default-transactions.io

```diff
@@ -0,0 +1,2 @@
+>> {"jsonrpc":"2.0","id":31,"method":"txpool_transactions"}
```

Comment:
So I think what we should here for the tests is have two `eth_sendRawTransaction`s before `txpool_transactions` so it _actually_ seeds the pool with the txs. This way we can still use these tests in hive to verify the response of each client. You can do this by defining the interaction as a test generator here: https://github.com/lightclient/rpctestgen/blob/main/testgen/generators.go